source/_integrations/zwave_js.markdown

```diff
@@ -433,7 +433,15 @@ Controlling your Z-Wave network using the Z-Wave JS integration has the followin
 
 Z-Wave JS is know to be compatible with all known Z-Wave Series 500 (Gen5) and Z-Wave Series 700 (Gen7) adapters in USB stick/dongle and serial module formats.
 
-The following devices have been confirmed to work with Z-Wave JS:
+It is recommend that you purchase a [Z-Wave Plus](https://z-wavealliance.org/z-wave_plus_certification/) compatible controller, to take advantage of the improvements this provides. As Z-Wave JS does not support the S2 security specifications, as such there is no need today to buy one just for support of the S2 feature, however that support for the S2 security specifications could possible be added to Z-Wave JS in the future (more infomation [here](https://github.com/zwave-js/node-zwave-js/projects/4)).
```

Comment:
This text does not add anything? I don't think we should link to a work in progress or even give a recommendation for a reason we don't support.